antctl supportbundle doesn't work on Windows Nodes

**Describe the bug**
A Failure is got when collecting support bundle on Windows Nodes.

**To Reproduce**
1. Download antctl binary on Windows Nodes
2. Run antctl supportbundle to collect Agent logs on the Nodes.

**Expected**
supportbundle file should be downloaded to the given path.

**Actual behavior**
Fail to generate supportbundle files, and a failure is printed:
```
E0825 10:58:27.163279    9080 rest.go:152] Error when collecting supportBundle: error when creating output tarfile: open C:\Windows\TEMP\bundle_*.tar.gz050868594: The filename, directory name, or volume label syntax is incorrect.
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x148 pc=0x18d2c52]
```


**Versions:**
Please provide the following information:
 - Antrea version (Docker image tag): 0.9.0/0.9.1

--- a/go.mod
+++ b/go.mod
@@ -32,7 +32,7 @@ require (
 	github.com/rakelkar/gonetsh v0.0.0-20190930180311-e5c5ffe4bdf0
 	github.com/satori/go.uuid v1.2.0
 	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/afero v1.2.2
+	github.com/spf13/afero v1.3.4
 	github.com/spf13/cobra v0.0.5
 	github.com/spf13/pflag v1.0.5
 	github.com/streamrail/concurrent-map v0.0.0-20160823150647-8bf1e9bacbf6 // indirect

